frontend/packages/eslint-plugin-console/lib/config/rules/react.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
   // One JSX element Per line
   'react/jsx-one-expression-per-line': 'off',
 
+  // Prop spreading is forbidden
+  'react/jsx-props-no-spreading': 'off',
```

Comment:
+1, as handy as this seems to be initially...

Reply:
Spread certainly has its usefulness. Especially when creating a component library. But often being explicit in the app components would be preferred.